README.md

```diff
@@ -56,14 +56,16 @@ expressOasGenerator.init(
   function(spec) { return spec; },
   'path/to/a/file/filename.json',
   60 * 1000,
-  ['User','Student']
+  ['User','Student'],
+  ['users', 'students']
 )
 ```
 where:
 * 'path/to/a/file/filename.json' - path to a file and file name
 * 60 * 1000 - write interval in milliseconds (optional parameter, by default interval is equal to 10 seconds)
 * ['User','Student'] - Mongoose models to be included as definitions. To get all just do mongoose.modelNames().
 The following peer dependencies are required to use this last argument: mongoose, mongoose-to-swagger, bson.
+* ['users', 'students'] - Tags to be used. Routes are grouped together by containing tags.
```

Comment:
I think this needs some further explanation, especially for those who might not know what tags are, but could get a good use out of them.

The two comments [1][2] @ the feature request issue helped me understand things way better, but ain't nobody going to look there, so some further yet concise explanation would be great!

[1] https://github.com/mpashkovskiy/express-oas-generator/issues/47#issuecomment-662799143

[2] https://github.com/mpashkovskiy/express-oas-generator/issues/47#issuecomment-664517658

Reply:
Updated readme to use clearer comments 👍